utils/video.js

```diff
@@ -33,7 +33,13 @@ export function getVimeoThumbnail(videoUrl) {
   const videoId = getVimeoId(videoUrl)
   return fetch(`https://vimeo.com/api/v2/video/${videoId}.json`)
     .then(response => response.json())
-    .then(videoData => videoData[0].thumbnail_medium)
+    .then((videoData) => {
+      if (videoData && videoData.length === 1) {
```

Comment:
Wouldn't `videoData.length > 0` be a better check? Just in case.

Reply:
I mean why else would it be an array, if it only ever has one entry. :)

Reply:
Good point. I will change it the way you suggested. And will add extra check to see if it is actually an `Array` since string has `length` property as well.